libs/contact/src/contact.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('contact library', () => {
```

Comment:
Yes you're totally right, this file could be removed in this PR. I initially created it with the creation of the library so that the testing suite would run and not error.